tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -1454,4 +1454,44 @@ public function testTypesContainingWhitespace(): void
              */
         ');
     }
+
+    public function testTypesContainingBackslash(): void
+    {
+        $this->doTest('<?php
+            /**
+             * @var string                            $input
+             * @var \Closure                          $fn
+             * @var \Closure(bool):int                $fn2
+             * @var Closure                           $fn3
+             * @var Closure(string):string            $fn4
+             * @var array<string,array<string,mixed>> $data
+             */
+            /**
+             * @param string                            $input
+             * @param \Closure                          $fn
+             * @param \Closure(bool):int                $fn2
+             * @param Closure                           $fn3
+             * @param Closure(string):string            $fn4
+             * @param array<string,array<string,mixed>> $data
+             */
+            /**
+             * @var string                   $value
+             * @var \Closure(string): string $callback
+             * @var Closure(int): bool       $callback2
+             */
+            /**
+             * @param string                   $value
+             * @param \Closure(string): string $callback
+             * @param Closure(int): bool       $callback2
+             */
+            /**
+             * @var Closure(array<int,bool>): bool $callback1
+             * @var \Closure(string): string       $callback2
+             */
+            /**
```

Comment:
I think all of those can be combined to one `/**` for `@var` and one for `@param`

Reply:
They are purposefully separate as sometimes the closure is the longest definition, sometimes it isn't.